File: level_2/find_unique_array/langs/compiler.cpp
int main()
{
    int n;
    string str;
    cin >> n;
    int arr[n];
    for (int i=0; i<n; i++)
        cin>>arr[i];
    
    find_unique_array(arr, n, &str);
    
    cout<<str;
    
    return 0;
}

File: level_2/narrative6_find_unique_array/langs/compiler.java
class Main
{
    public static void main(String[] args) 
    {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] arr = new int[n];
        for (int i = 0; i < n; i++)
        arr[i] = scanner.nextInt();
        
        Test test = new Main();
        test.find_unique_array(arr, n);
        String str = test.result;  
        System.out.println(str);
        scanner.close();
    }
}